pkg/cmd/pipelineresource/list.go

```diff
@@ -81,13 +83,21 @@ func listCommand(p cli.Params) *cobra.Command {
 				return fmt.Errorf("failed to list pipelineresources. Invalid resource type %s", opts.Type)
 			}
 
-			pres, err := list(cs.Resource, p.Namespace(), opts.Type)
+			namespace := p.Namespace()
+			if opts.AllNamespaces {
+				namespace = ""
+			}
+			pres, err := list(cs.Resource, namespace, opts.Type)
 			stream := &cli.Stream{
 				Out: cmd.OutOrStdout(),
 				Err: cmd.OutOrStderr(),
 			}
 			if err != nil {
-				fmt.Fprintf(os.Stderr, "Failed to list pipelineresources from %s namespace \n", p.Namespace())
+				ns := namespace
+				if opts.AllNamespaces {
+					ns = "all"
+				}
+				fmt.Fprintf(os.Stderr, "Failed to list pipelineresources from %s namespace \n", ns)
```

Comment:
```suggestion
				fmt.Fprintf(os.Stderr, "Failed to list pipelineresources from %s namespaces \n", ns)
```

Reply:
I think `namespace(s)` would be better. Should I use this instead ?

Reply:
Yup, good catch. Makes sense to me.